src/corva/handlers.py

```diff
@@ -20,8 +24,9 @@
 from corva.service.api_sdk import CachingApiSdk, CorvaApiSdk
 from corva.service.cache_sdk import FakeInternalCacheSdk, InternalRedisSdk, UserRedisSdk
 
-StreamEventT = TypeVar('StreamEventT', bound=StreamEvent)
-ScheduledEventT = TypeVar('ScheduledEventT', bound=ScheduledEvent)
+StreamEventT = TypeVar("StreamEventT", bound=StreamEvent)
+ScheduledEventT = TypeVar("ScheduledEventT", bound=ScheduledEvent)
+HANDLERS: Dict[Optional[Type[Union[RawPartialRerunMergeEvent]]], Callable] = {}
```

Comment:
Maybe `RawBaseEvent` will make code more generic and simpler to use
```suggestion
HANDLERS: Dict[Type[RawBaseEvent], Callable] = {}
```

Reply:
This is addressed.